### PR TITLE
fix: make logs better

### PR DESCRIPTION
**Pull request by parishahla**
The deploying state in showReleaseLogs() is not applicable. It's an ongoing process, hence keeps logging to the console.

### CODE REVIEW

File: src/commands/deploy.ts [hunk 1]
```diff
@@ -643,15 +643,15 @@ Additionally, you can also retry the build with the debug flag:
...
+        this.debug('No version for this platform found. Using default version');
       }
     }
 
     return body;
   }
 
   async showReleaseLogs(releaseID: string) {
-    this.spinner.start('Creating a new release...');
+    this.spinner.start('Checking container health...');
```
Comment (eyeamnoob) [inline src/commands/deploy.ts:654]: It should remains the same as it was. We don't check health here.

**Comment by parishahla** [comment]
> It's an ongoing process, hence keeps logging to the console.

As I mentioned earlier, if the process takes longer to finish, it will keep logging it to the console. I brought it back though.

**Review by eyeamnoob — COMMENTED** [review]
GREAT

**Comment by eyeamnoob** [comment]
> > It's an ongoing process, hence keeps logging to the console.
> 
> As I mentioned earlier, if the process takes longer to finish, it will keep logging it to the console. I brought it back though.

Is this bug still around? W checked together and saw nothing.

**Comment by parishahla** [comment]
> > > It's an ongoing process, hence keeps logging to the console.
> > 
> > 
> > As I mentioned earlier, if the process takes longer to finish, it will keep logging it to the console. I brought it back though.
> 
> Is this bug still around? W checked together and saw nothing.

It is. I released the recent changes except the showBuildLogs() > DEPLOYING stage, till we make sure that it's gonna be working properly and make the necessary changes.

**Comment by mhemrg** [comment]
What is the state of this?

**Comment by mhemrg** [comment]
This LGTM! What do you think @eyeamnoob?

**Comment by eyeamnoob** [comment]
I think we don't have any problem in showing logs. I close this PR.

**Pull request closed without merging**